src/odemis/gui/cont/tabs/sparc2_align_tab.py

```diff
@@ -274,6 +274,13 @@ def __init__(self, name, button, panel, main_frame, main_data):
 
         ccd_focuser_ext = None
         if "tunnel-lens-align" in tab_data.align_mode.choices:
+            # if there is a favourite position, go to that position at init
+            if main_data.spec_ded_aligner:
+                md = main_data.spec_ded_aligner.getMetadata()
+                if model.MD_FAV_POS_ACTIVE in md:
```

Comment:
How is this favorite position set?

Reply:
Indeed, that's set in the microscope file, based on the alignment done during installation.
The user has the possibility to adjust it afterwards from the alignment tab... At least in theory, because we've just noticed this part doesn't work!